cookbook/specs/spec_add_manta.json

```diff
@@ -0,0 +1,115 @@
+{
+    "proposal": {
+      "title": "Add Specs: Manta",
+      "description": "Adding new specification support for relaying data on Manta",
+      "specs": [
+        {
+          "index": "MANTA",
+          "name": "manta mainnet",
+          "enabled": true,
+          "imports": ["ETH1"],
+          "reliability_threshold": 268435455,
+          "data_reliability_enabled": true,
+          "block_distance_for_finalized_data": 1,
+          "blocks_in_finalization_proof": 3,
+          "average_block_time": 10000,
+          "allowed_block_lag_for_qos_sync": 5,
+          "shares": 1,
+          "min_stake_provider": {
+            "denom": "ulava",
+            "amount": "47500000000"
+          },
+          "api_collections": [
+            {
+              "enabled": true,
+              "collection_data": {
+                "api_interface": "jsonrpc",
+                "internal_path": "",
+                "type": "POST",
+                "add_on": ""
+              },
+              "apis": [],
+              "headers": [],
+              "inheritance_apis": [],
+              "parse_directives": [],
+              "verifications": [
+                {
+                  "name": "chain-id",
+                  "values": [
+                    {
+                      "expected_value": "0xa9"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 0
+                    },
+                    {
+                      "extension": "archive",
+                      "expected_value": "0x0"
+                    }
+                  ]
+                }
+              ],
+              "extensions": [
+                {
+                  "name": "archive",
+                  "cu_multiplier": 5,
+                  "rule": {
+                    "block": 0
```

Comment:
needs to be set otherwise archive won't work, rule of thumb is a 24h - 15m

Reply:
just to clarify, @omerlavanet means that the value that you need to put is:
<manta_average_block_time_in_seconds> * <number_of_seconds_in_23H_and_45M> = number of blocks in 23H and 45M

Reply:
I added 8550 
10 seconds average a block meaning 8550 block on 24H-15MIN